### DISCUSSION TITLE
Windows Docker container connection

### DISCUSSION THREAD

**Discussion by Liripo** [posit-dev/positron#4684] [Q&A] [answered]
Does anyone have an example of connecting to a docker container on windows?

**Comment by petetronic** [comment]
Hi @Liripo, could you share a little more about the scenario you're trying to set up?

**Comment by Liripo** [comment]
@petetronic  I typically use R in a Windows Docker container, and in VSCode, I can directly connect to existing containers within the IDE.for example:
![image](https://github.com/user-attachments/assets/e7bbea24-a9b6-4c45-8fb8-48655626e150)

**Reply by jmcphers** [reply, marked as answer]
The Dev Containers feature is a Microsoft proprietary extension to VS Code, so it is not currently part of Positron. I wrote up an issue here: https://github.com/posit-dev/positron/issues/4691
